README.md

```diff
@@ -157,6 +157,24 @@ bench.concurrency = 'task' // The concurrency mode to determine how tasks are ru
 await bench.run()
 ```
 
+## Retaining Samples
+
+By default, Tinybench does not retain individual samples (latency measurements) for each task to save memory.
```

Comment:
The samples property is expected to be here for latency and throughput distribution for consistency: the later is computed from the former but recomputation done elsewhere is: 
+ error prone
+ not stable numerically

To ensure advanced statistical distribution comparison can be used (KL divergence, Wasserstein distance, ...), a single consistent source of truth for samples is required. 

Tinybench offer some widely known default statistical estimators not computationally intensive and should allow to be plugged to advanced performance analysis tools that will gather the samples over time and allow in depth analysis of the performance: impact of the runtime version, impact of a single commit, ... using advanced statistical estimators (correlation, distribution comparison, ...)